src/assets-list/assets-list.js

```diff
@@ -0,0 +1,91 @@
+import jQuery from 'jquery';
+import template from './assets-list.pug';
+const assetUrl= 'http://fuego-app.herokuapp.com/api/assets';
+import Assets from '../services/assets';
+
+const component = jQuery('<div class="assets-list-component component"></div>');
```

Comment:
never run code that is outside a function or class (unless this is the app.js file, which servers to initialize the whole app).